package.json

```diff
@@ -67,19 +68,21 @@
     "run": "web-ext run -s dist --firefox=nightly",
     "clean": "rm -rf dist && shx mkdir -p dist/popup",
     "watch": "npm-run-all --parallel watch:*",
-    "copy:locales": "node pontoon-to-webext",
-    "copy:assets": "shx cp -r src/manifest.json src/icons dist/ && svgo dist/icons --quiet && shx cp -r src/popup/*.html src/popup/FiraSans-Regular.* dist/popup/",
+    "copy:locales": "node ./bin/pontoon-to-webext",
+    "copy:assets": "shx cp -r src/icons dist/ && svgo dist/icons --quiet && shx cp -r src/popup/*.html src/popup/FiraSans-Regular.* dist/popup/",
     "copy:js": "shx cp node_modules/testpilot-metrics/testpilot-metrics.js dist/",
+    "bundle:manifest": "node ./bin/generate-manifest",
     "bundle:js": "npm run lint:js && rollup -c --environment entry:background && rollup -c --environment entry:popup/snooze-content && rollup -c --environment entry:lib/confirm-bar",
     "bundle:css": "npm run lint:sass && shx mkdir -p dist/popup && node-sass src/popup/snooze.scss > dist/popup/snooze.css",
     "package": "npm run build && web-ext build -s dist -a .",
     "lint": "npm-run-all --parallel lint:*",
     "lint:js": "eslint --max-warnings=0 .",
     "lint:sass": "sass-lint --max-warnings 0 --verbose --no-exit",
+    "watch:manifest": "onchange -p -v 'bin/generate-manifest.js' -- npm run bundle:manifest",
     "watch:js": "onchange -p -v 'src/**/*.js' -- npm-run-all --parallel bundle:js test:js",
     "watch:test:js": "onchange -p -v 'test/**/*.js' -- npm-run-all --parallel test:js",
     "watch:css": "onchange -p -v 'src/**/*.scss' -- npm run bundle:css",
-    "watch:assets": "onchange -p -v 'src/{manifest.json,icons/*,popup/*.html}' -- npm run copy:assets",
+    "watch:assets": "onchange -p -v 'src/{icons/*,popup/*.html}' -- npm run copy:assets",
```

Comment:
If we had a `manifest.in` we could watch that, but since we don't (yet), should we watch `bin/generate-manifest` for changes to permissions and stuff, too?